activeadmin.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'arbre', '>= 1.1.1'
   s.add_dependency 'formtastic', '~> 3.1'
   s.add_dependency 'formtastic_i18n'
+  s.add_dependency 'activemodel-serializers-xml'
```

Comment:
Can you add `~> 1.0` here, so that we never unintentionally allow a potentially breaking release?